### PR TITLE
Label printer partial back end

### PR DESCRIPTION
**Pull request by iaincrawf**
enough to build the front end but haven't done the actual printing bit yet

### CODE REVIEW

File: src/Facade/ResourceBuilders/IdAndNameResourceBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            {
+                Id = idAndName.Id,
+                Name = idAndName.Name,
+                Links = this.BuildLinks(idAndName).ToArray()
+            };
+        }
+
+        public string GetLocation(IdAndName idAndName)
+        {
+            return string.Empty;
```
Comment (lewisrenfrew) [inline src/Facade/ResourceBuilders/IdAndNameResourceBuilder.cs]: Is there a reason location should be an empty string? Could be misleading if not. Could throw a NotImplementedException to make it clearer.

File: src/Facade/ResourceBuilders/LabelPrintResourceBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    using Linn.Common.Authorisation;
+    using Linn.Common.Facade;
+    using Linn.Common.Resources;
+    using Linn.Production.Domain.LinnApps;
+    using Linn.Production.Resources;
+
+    public class LabelPrintResourceBuilder : IResourceBuilder<LabelPrint>
+    {
+        public LabelPrintResourceBuilder()
```
Comment (lewisrenfrew) [inline src/Facade/ResourceBuilders/LabelPrintResourceBuilder.cs]: Is there something making the empty constructor mandatory here?

File: src/Facade/Services/LabelPrintService.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
+﻿namespace Linn.Production.Facade.Services
+
```
Comment (lewisrenfrew) [inline src/Facade/Services/LabelPrintService.cs]: blank

File: tests/Integration/Service.Tests/LabelPrintModuleSpecs/ContextBase.cs [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class ContextBase : NancyContextBase
+    {
+        protected ILabelPrintService LabelPrintService { get; private set; }
+        
+    [SetUp]
+        public void EstablishContext()
```
Comment (lewisrenfrew) [inline tests/Integration/Service.Tests/LabelPrintModuleSpecs/ContextBase.cs]: weird indenation

File: src/Facade/ResourceBuilders/IdAndNameResourceBuilder.cs (hunk 1 above)
Comment (iaincrawf) [reply src/Facade/ResourceBuilders/IdAndNameResourceBuilder.cs]: Nope good point, thinking about it I probably don't actually need to return any links with empty Hrefs either, think I'd be better just returning an empty list of Link Resources

**Pull request merged**